addons/account/report/account_tax_details_report.py

```diff
@@ -0,0 +1,385 @@
+        string="Source Base Journal Item",
+        readonly=True,
+        required=True,
+    )
+    tax_id = fields.Many2one(
+        comodel_name='account.tax',
+        string="Originator Tax",
+        readonly=True,
+        required=True,
+    )
+    group_tax_id = fields.Many2one(
+        comodel_name='account.tax',
+        string="Originator Group of Taxes",
+        readonly=True,
+        required=True,
+    )
+    base_amount = fields.Monetary(
+        string="Base Amount",
+        readonly=True,
+        currency_field='company_currency_id',
+    )
+    tax_amount = fields.Monetary(
+        string="Tax Amount",
+        readonly=True,
+        currency_field='company_currency_id',
+    )
+    base_amount_currency = fields.Monetary(
+        string="Base Amount Currency",
+        readonly=True,
+        currency_field='currency_id',
+    )
+    tax_amount_currency = fields.Monetary(
+        string="Tax Amount Currency",
+        readonly=True,
+        currency_field='currency_id',
+    )
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string="Company",
+    )
+    company_currency_id = fields.Many2one(
+        comodel_name='res.currency',
+    )
+    currency_id = fields.Many2one(
+        comodel_name='res.currency',
+    )
+
+    def init(self):
+        self._cr.execute('''
+            CREATE OR REPLACE VIEW account_tax_details_report AS (
+
+                WITH tax_ids_m2m AS (
+
+                    SELECT
+                        tax_rel.account_move_line_id AS id,
+                        ARRAY_AGG(tax_rel.account_tax_id ORDER BY tax.sequence, tax.id) AS tax_ids
+                    FROM account_move_line_account_tax_rel tax_rel
+                    JOIN account_move_line base_line ON base_line.id = tax_rel.account_move_line_id
+                    JOIN account_tax tax ON tax.id = tax_rel.account_tax_id
+                    WHERE tax.amount_type != 'group'
+                        AND (tax.is_base_affected OR base_line.tax_repartition_line_id IS NOT NULL)
+                    GROUP BY tax_rel.account_move_line_id
+
+                    UNION ALL
+
+                    SELECT
+                        tax_rel.account_move_line_id AS id,
+                        ARRAY_AGG(group_tax_rel.child_tax ORDER BY tax.sequence, tax.id) AS tax_ids
+                    FROM account_move_line_account_tax_rel tax_rel
+                    JOIN account_tax tax ON tax.id = tax_rel.account_tax_id
+                    JOIN account_move_line base_line ON base_line.id = tax_rel.account_move_line_id
+                    JOIN account_tax_filiation_rel group_tax_rel ON group_tax_rel.parent_tax = tax.id
+                    WHERE tax.amount_type = 'group'
+                        AND (tax.is_base_affected OR base_line.tax_repartition_line_id IS NOT NULL)
+                    GROUP BY tax_rel.account_move_line_id
+
+                ),
+
+                base_tax_matching_no_tax_line_as_base AS (
```

Comment:
meaningless name (to me): what does this CTE achieves should be transpiring in its name